forge-gui/res/cardsfolder/upcoming/star_athlete.txt

```diff
@@ -0,0 +1,12 @@
+Name:Star Athlete
+ManaCost:2 R R
+Types:Creature Human Warrior
+PT:3/2
+K:Menace
+K:Blitz:3 R
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBPump | TriggerDescription$ Whenever Star Athlete attacks, choose up to one target nonland permanent. Its controller may sacrifice it. If they don't, Star Athlete deals 5 damage to that player.
+SVar:DBPump:DB$ Pump | ValidTgts$ Permanent.nonLand | RememberTargets$ True | TgtPrompt$ Select up to one target nonland permanent | SubAbility$ DBDamage

```

Comment:
the "up to" part isn't handled yet

Reply:
Addressed in [commit](https://github.com/Card-Forge/forge/pull/6225/commits/3e263a0c8be3cd761664454c4befac2372805eae)